### PR TITLE
P1 loading roast

### PR DESCRIPTION
**Pull request by dev-prabhat**
Check these files
https://github.com/dev-prabhat/super-flex-plus/pull/6/files?file-filters%5B%5D=.css&file-filters%5B%5D=.js&show-viewed-files=true

### CODE REVIEW

File: src/customHooks/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { useAxios } from "./useAxios";
+import { useDoc } from "./useDoc";
```
Comment (sudipkundu999) [inline src/customHooks/index.js]: suggestion: 
rename `useDoc` to `useDocumentTitle`
The Naming of the hook should explain what the hook does.

**Review by sudipkundu999 — APPROVED** [review]
Good work! Good to merge

**Review by Dev-AkshatGupta — APPROVED** [review]
I found everything to be ok . You are good to go and can merge it

File: src/components/Loading/Loading.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    const {playlistLoading, playlistVideoLoading,} = usePlaylist()
+    
+    if(!likeApiCallLoading && !watchLaterApiCallLoading && !isLoading && !playlistLoading && !playlistVideoLoading) return null
+    return(
+        <div className='loading-wrapper'>
+            {likeApiCallLoading && <ReactLoading className='reacting-loading' type={"bars"} color={"#ffa500"}/>}
+            {watchLaterApiCallLoading && <ReactLoading className='reacting-loading' type={"bars"} color={"#ffa500"}/>}
+            {isLoading && <ReactLoading className='reacting-loading' type={"bars"} color={"#ffa500"}/>}
+            {playlistLoading && <ReactLoading className='reacting-loading' type={"bars"} color={"#ffa500"}/>}
+            {playlistVideoLoading && <ReactLoading className='reacting-loading' type={"bars"} color={"#ffa500"}/>}
```
Comment (iRohitGaur) [inline src/components/Loading/Loading.js:17]: suggestion: you can use logical OR here with the loader vars

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Good to merge 👍🏻

**Pull request merged**